EASTR/alignment_utils.py

```diff
@@ -46,27 +43,27 @@ def calc_alignment_score(hit,scoring):
     matches = hit.mlen
     gap_penalty = 0
     cs = hit.cs
-    
+
     #gaps
     p = re.compile('[\\-\\+]([atgc]+)')
     m = p.findall(cs)
     gaps = len(m)
     for gap in m:
         gap_len = len(gap)
         gap_penalty += min(scoring[2] + (gap_len - 1) * scoring[3],
-                            scoring[4] + (gap_len - 1) * scoring[5])  
-        
+                            scoring[4] + (gap_len - 1) * scoring[5])
+
     #mismatches
     p = re.compile('\\*([atgc]+)')
     m = p.findall(cs)
     mismatches = len(m)
-        
-    AS = matches*scoring[0] - (mismatches)*scoring[1] - gap_penalty
```

Comment:
this variable is named 'AS' because it means alignment score.

Reply:
sounds good.

Reply:
Done. See [69bfa28db225222a2480fadd1368b0a4c013e6b3](https://github.com/ishinder/EASTR/pull/16/commits/69bfa28db225222a2480fadd1368b0a4c013e6b3)